odoo100/scripts/10.0-full_requirements.txt

```diff
@@ -66,3 +66,4 @@ wsgiref==0.1.2
 pyparsing==2.1.10
 git+https://github.com/vauxoo/pylint-odoo@master#egg=pylint-odoo
 git+https://github.com/vauxoo/panama-dv@master#egg=ruc
+html2text==2016.9.19
```

Comment:
@ruiztulio 
Sorry, this change not affect the odoo-10.0 version.
Only the master version required this package https://github.com/odoo/odoo/blob/master/requirements.txt#L9

Is it enough by appending in odoo-shippable/scripts/build-image.sh?